### ISSUE TITLE
input data format

### ISSUE THREAD

**Issue by gianfilippo** [jerryji1993/DNABERT#5]
Hi,

could you please explain me what is the label (0,1) in the input data ?

I want to fine tune the model with the sequences I have and use the embeddings for downstream processing

Thanks

**Comment by Zhihan1996** [comment]
Currently, there are only two possible choices for MODEL_TYPE, ```dna``` and ```dnalongcat```. The ```dna``` is used for sequences shorter than 512 and the second one is used for longer sequences. When using ```dnalongcat```, please make sure the sequence length equals to N*512 (e.g., 1024, 2048).

The TASK_NAME is used for the code the understand what task you want to perform and how the data should be load. For 2-class classification with your own data, you can process the data to the format as the sample data and set TASK_NAME as dnaprom. For other cases, you need to define your own task, which is more complicated. 
First, go to ```src/transformers/data/processors.py``` to define a new processor class for your own data (including the number of labels and how to load data). Second, go to the bottom of this file to update these three dictionaries, which is straightforward. Third, go to the ```src/transformers/data/metrics/__init__.py``` line 98 to define the metric (e.g., f1/accuracy) you want to calculate for your task.

**Comment by gianfilippo** [comment]
Thanks for explaining.
Are the available pretrained models of MODEL_TYPE dna or dnalongcat ?
If they are of MODEL_TYPE dna, do you have a pretrained model of
MODEL_TYPE dnalongcat ?
Thanks

**Comment by Zhihan1996** [comment]
Sure. The pre-trained models work for both dna and dnalongcat. Please directly use them.

**Comment by gianfilippo** [comment]
ok. Thanks

**Comment by gianfilippo** [comment]
Hi,

about your suggestions to modify the code, I have a few questions:
1) the src/transformers/data/processors.py does not exist or I cannot see it. Do I have to create it ?
2) what three directories ? at the bottom of which file ? processors.py ?

Thanks

**Comment by aliakay** [comment]
Hi, How can I fintune the model for regression cases?
